src/views/producer/call-details/components/InjectionStreamCard.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import React, { useState } from "react";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
```

Comment:
**nitpick(blocking)**: there is an unused import (useSelector)

Reply:
Hi, thanks for the recommendation, it has already been applied.